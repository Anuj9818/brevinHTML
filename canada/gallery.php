<?php
require 'config.php';
require 'dbconnect.php';

include 'head.php';
$selectedMenu="gallery";
include 'header.php';
?>

<section id="subpage-top">
	<div class="container">
		<h1>Gallery</h1>
	</div>
</section>

<section id="subpage-wrap">
	<div class="container clearfix">
		<div id="subpage-full" class="post">
			<div id="content">
		
			<?php
				
				$id=intval($_GET["id"]);
				$page=intval($_GET["page"]);
				$linkedPage="gallery.php";
				
				if ($id=="0") {
					$getLatestGallery=mysqli_fetch_array(mysqli_query($con, "select * from gallery_name order by posted_date desc, id desc limit 1"));
					$id=$getLatestGallery['id'];
				}

				$getDataQuery=mysqli_query($con, "select * from gallery_name where id='$id'");								
				if(mysqli_num_rows($getDataQuery)==0) {
					print "Error! Data doesn't exist.";
				} else {
					$eachData=mysqli_fetch_array($getDataQuery);
					$dataTitle=html_entity_decode($eachData['title']);
					$dataDetails=html_entity_decode($eachData['details']);
					$dataDate=date("M d, Y", strtotime($eachData['posted_date']));

					print "<h3>$dataTitle</h3>";
					print "<div>Date: $dataDate</div>";
					print "<div>$dataDetails</div>";

					print "<div style=\"margin-top: 0px;\">";
					$getPhotos=mysqli_query($con, "select * from gallery_image where gallery_id='$id' order by id desc");
					if (mysqli_num_rows($getPhotos)>0) {
						print "<ul class='gallery' style='list-style-type:none; margin:15px 0; padding:0px;'>";

						while ($eachPhotos=mysqli_fetch_array($getPhotos)) {
							$dataImg=$eachPhotos['img_name'];
							print "<li>
								<a href=\"$common_path/uploads/gallery/$dataImg\" data-lightbox='roadtrip' title=\"$dataTitle\" class=\"photolink\"><img src='$common_path/uploads/gallery/$dataImg' border='0' alt='$title'></a>
							</li>";
						}
						print "</ul>";
					} else {
						print "<div class=\"txt\" style=\"padding: 20px; color: #d51818;\">Photos doesn't exist.</div>";
					}
					print "</div>";
				}
				?>

				<br class='clear'><br><br>

				<div class="txt" style="padding:0 0 10px 10px;">More Albums</div>

				<?php
				$limit=10;
				
				$query1="select * from gallery_name";
				$result1=mysqli_query($con, $query1);
				$totalrows=mysqli_num_rows($result1);

				if ($totalrows==0) {
					print "ERROR! Albums Doesn't Exist.";
				} else {
					if(empty($page)) {
						$page = 1; 
					}
					$limitvalue1 = $page*$limit-($limit);

					$getAlbumsQueryData=$query1." order by posted_date desc, id desc LIMIT $limitvalue1, $limit";
					$getAlbumsQuery=mysqli_query($con, $getAlbumsQueryData);

					while ($eachData=mysqli_fetch_array($getAlbumsQuery)) {
						$altRow++;
						$albumID=$eachData['id'];
						$albumTitle=html_entity_decode($eachData['title']);
						$albumDetails=html_entity_decode(stripslashes($eachData['details']));
						$albumShortDetails=substr(strip_tags($albumDetails), 0, 160);
						
						print "<div style='padding:0 0 0 10px;'>
							<a href='gallery.php?id=$albumID&page=$page' class='titleMenu'>$albumTitle</a>
							<div>$albumShortDetails ...</div>
						</div>
						<hr class='thin'>";
					}

					$raw_pages=$totalrows/$limit;
					$int_pages=(integer) $raw_pages;
					if($raw_pages>$int_pages) {
						$numofpages=$int_pages+1;
					} else {
						$numofpages=$raw_pages;
					}
					if($page!=1) {
						$pageprev=$page-1;
						$prevCodeOutput="<li><a href=\"gallery.php?id=$albumID&page=$pageprev\" class=\"prevnext\">&laquo; Previous</a></li>\n";
					} else {
						$prevCodeOutput="<li><a href=\"#\" class=\"prevnext disablelink\">&laquo; Previous</a></li>\n";
					}

					$loopPage="";
					for($i=1; $i<=$numofpages; $i++) {
						$loopPage.="<li><a href=\"gallery.php?id=$albumID&page=$i\"";
						if ($page==$i) {
							$loopPage.="class=\"currentpage\"";
						}
						$loopPage.=">$i</a></li>\n";
					}

					if($page<$numofpages && $numofpages>1){
						$pagenext=$page+1;
						$nextCodeOutput="<li><a href=\"gallery.php?id=$albumID&page=$pagenext\" class=\"prevnext\">Next &raquo;</a></li>\n";
					} else {
						$nextCodeOutput="<li><a href=\"#\" class=\"prevnext disablelink\">Next &raquo;</a></li>\n";
					}

					print "<div class=\"pagination\">
						<ul>
							$prevCodeOutput
							$loopPage
							$nextCodeOutput
						</ul>
					</div>";
				}
				?>

				<br><br>
			<div id="fb-root"></div>
		<script>(function(d, s, id) {
		  var js, fjs = d.getElementsByTagName(s)[0];
		  if (d.getElementById(id)) return;
		  js = d.createElement(s); js.id = id;
		  js.src = 'https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.10&appId=243738082480264';
		  fjs.parentNode.insertBefore(js, fjs);
		}(document, 'script', 'facebook-jssdk'));</script>

		<div class="fb-page" data-href="https://www.facebook.com/canconsultancee/" data-tabs="timeline" data-width="500" data-small-header="true" data-adapt-container-width="true" data-hide-cover="true" data-show-facepile="false"><blockquote cite="https://www.facebook.com/canconsultancee/" class="fb-xfbml-parse-ignore"><a href="https://www.facebook.com/canconsultancee/">Can Consultancy</a></blockquote></div>
		</div>
		
	</div>
</section>

<?php
include 'footer.php';
?>



