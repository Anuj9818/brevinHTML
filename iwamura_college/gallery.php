<?php
require 'config.php';
require 'dbconnect.php';
require ('inc/function.php');
include 'head.php';
$selectedMenu="news";
include 'header.php';
include 'inc/notice-board.php';
?>

<div class="content-wrap">
	<div class="content">
		<div class="subpage">
			<h1>Photo Gallery</h1>
			<?php
				
				$id=intval($_GET["id"]);
				$page=intval($_GET["page"]);
				$linkedPage="gallery.php";
				
				if ($id=="0") {
					$getLatestGallery=mysqli_fetch_array(mysqli_query($con, "select * from gallery_name order by posted_date desc, id desc limit 1"));
					$id=$getLatestGallery['id'];
				}

				$getDataQuery=mysqli_query($con, "select * from gallery_name where id='$id'");								
				if(mysqli_num_rows($getDataQuery)==0) {
					print "Error! Data doesn't exist.";
				} else {
					$eachData=mysqli_fetch_array($getDataQuery);
					$dataTitle=html_entity_decode($eachData['title']);
					$dataDetails=html_entity_decode($eachData['details']);
					$dataDate=date("M d, Y", strtotime($eachData['posted_date']));

					print "<h3>$dataTitle</h3>";
					print "<div class=\"txt\" style=\"padding:0 0 0 0px;\">Date: $dataDate</div>";
					print "<div class=\"txt\" style=\"padding:0 0 0 0px;\">$dataDetails</div>";

					print "<div style=\"margin-top: 0px;\">";
					$getPhotos=mysqli_query($con, "select * from gallery_image where gallery_id='$id' order by id desc");
					if (mysqli_num_rows($getPhotos)>0) {
						print "<ul class='gallery-list' style='list-style-type:none; margin:15px 0; padding:0px;'>";

						while ($eachPhotos=mysqli_fetch_array($getPhotos)) {
							$dataImg=$eachPhotos['img_name'];
							print "<li>
								<a href=\"$common_path/uploads/gallery/$dataImg\" data-lightbox='roadtrip' title=\"$dataTitle\"><img src='$common_path/uploads/gallery/$dataImg' width='140' border='0' alt='$title'></a>
							</li>";
						}
						print "</ul>";
					} else {
						print "<div class=\"txt\" style=\"padding: 20px; color: #d51818;\">Photos doesn't exist.</div>";
					}
					print "</div>";
				}
				?>

				<br class='clear'><br><br>

				<div class="txt" style="padding:0 0 10px 10px;">More Albums</div>

				<?php
				$limit=10;
				
				$query1="select * from gallery_name";
				$result1=mysqli_query($con, $query1);
				$totalrows=mysqli_num_rows($result1);

				if ($totalrows==0) {
					print "ERROR! Albums Doesn't Exist.";
				} else {
					if(empty($page)) {
						$page = 1; 
					}
					$limitvalue1 = $page*$limit-($limit);

					$getAlbumsQueryData=$query1." order by posted_date desc, id desc LIMIT $limitvalue1, $limit";
					$getAlbumsQuery=mysqli_query($con, $getAlbumsQueryData);

					while ($eachData=mysqli_fetch_array($getAlbumsQuery)) {
						$altRow++;
						$albumID=$eachData['id'];
						$albumTitle=html_entity_decode($eachData['title']);
						$albumDetails=html_entity_decode(stripslashes($eachData['details']));
						$albumShortDetails=substr(strip_tags($albumDetails), 0, 160);
						
						print "<div style=\"padding:0 0 0 10px;\">
							<a href=\"gallery.php?id=$albumID&page=$page\" class=\"seeDetailMenu\">$albumTitle</a>
							<div class=\"txt\">$albumShortDetails ...</div>
						</div>
						<hr class='thin'>";
					}

					$raw_pages=$totalrows/$limit;
					$int_pages=(integer) $raw_pages;
					if($raw_pages>$int_pages) {
						$numofpages=$int_pages+1;
					} else {
						$numofpages=$raw_pages;
					}
					if($page!=1) {
						$pageprev=$page-1;
						$prevCodeOutput="<li><a href=\"gallery.php?id=$albumID&page=$pageprev\" class=\"prevnext\">&laquo; Previous</a></li>\n";
					} else {
						$prevCodeOutput="<li><a href=\"#\" class=\"prevnext disablelink\">&laquo; Previous</a></li>\n";
					}

					$loopPage="";
					for($i=1; $i<=$numofpages; $i++) {
						$loopPage.="<li><a href=\"gallery.php?id=$albumID&page=$i\"";
						if ($page==$i) {
							$loopPage.="class=\"currentpage\"";
						}
						$loopPage.=">$i</a></li>\n";
					}

					if($page<$numofpages && $numofpages>1){
						$pagenext=$page+1;
						$nextCodeOutput="<li><a href=\"gallery.php?id=$albumID&page=$pagenext\" class=\"prevnext\">Next &raquo;</a></li>\n";
					} else {
						$nextCodeOutput="<li><a href=\"#\" class=\"prevnext disablelink\">Next &raquo;</a></li>\n";
					}

					print "<div class=\"pagination\">
						<ul>
							$prevCodeOutput
							$loopPage
							$nextCodeOutput
						</ul>
					</div>";
				}
				?>
		</div>
		<?php include 'sidebar.php'; ?>
		
		<div class="clear"></div>
	</div>
</div>

<?php
include 'footer.php';
?>