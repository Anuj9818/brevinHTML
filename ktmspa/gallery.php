<?php include_once('header.php'); ?>

<!-- breadkcrumb wrapper starts here -->
<div class="breadcrumb-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <div class="col-lg-6 float-left">
                    <h2>Gallery</h2>
                </div>
                <div class="col-lg-6 float-left">
                    <h3><a href="./">Home </a> <i class="fa fa-angle-double-right" aria-hidden="true"></i> Gallery </h3>
                </div>
            </div>
        </div>
    </div>
</div>

<section class="content-wrapper section-padding">
    <div class="container">
        <div class="row">
            <ul class="galleryul">
                <li class="col-lg-4 col-md-4 col-sm-12 col-xs-12 float-left">
                    <a data-fancybox="gallery" href="images/gallery/1.jpg" data-caption="Ktm spa"><img src="images/gallery/1.jpg" alt=""></a>
                </li>
                 <li class="col-lg-4 col-md-4 col-sm-12 col-xs-12 float-left">
                    <a data-fancybox="gallery" href="images/gallery/2.jpg" data-caption="Ktm spa"><img src="images/gallery/2.jpg" alt=""></a>
                </li>
                 <li class="col-lg-4 col-md-4 col-sm-12 col-xs-12 float-left">
                    <a data-fancybox="gallery" href="images/gallery/3.jpg" data-caption="Ktm spa"><img src="images/gallery/3.jpg" alt=""></a>
                </li>
                 <li class="col-lg-4 col-md-4 col-sm-12 col-xs-12 float-left">
                    <a data-fancybox="gallery" href="images/gallery/4.jpg" data-caption="Ktm spa"><img src="images/gallery/4.jpg" alt=""></a>
                </li>
                 <li class="col-lg-4 col-md-4 col-sm-12 col-xs-12 float-left">
                    <a data-fancybox="gallery" href="images/gallery/5.jpg" data-caption="Ktm spa"><img src="images/gallery/5.jpg" alt=""></a>
                </li>
                 <li class="col-lg-4 col-md-4 col-sm-12 col-xs-12 float-left">
                    <a data-fancybox="gallery" href="images/gallery/6.jpg" data-caption="Ktm spa"><img src="images/gallery/6.jpg" alt=""></a>
                </li>
            </ul>
        </div>
    </div>
</section>

<?php include_once('footer.php'); ?>
