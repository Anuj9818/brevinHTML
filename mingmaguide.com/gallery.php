<?php include_once('header.php'); ?>

<!-- breadkcrumb wrapper starts here -->
<div class="breadcrumb-wrapper">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
            <h2>Gallery</h2>            
        </div>
        </div>
    </div>
</div>

<!-- Gallery Section -->
<section class="section-padding recents-section">
  <div class="container">
    <div class="row">
        <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
          <div class="recent-block">
            <img src="images/activity/1.jpg" class="img-responsive responsive--full wp-post-image" alt="Mingma Sherpa" /> 
            <div class="recent-content">
              <hr>
              <div class="clearfix"></div>
                <a href="media-gallery.php"><h2>Media Coverage</h2> </a>      
            </div>
          </div>
        </div>

        <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
          <div class="recent-block">
            <img src="images/activity/2.jpg" class="img-responsive responsive--full wp-post-image" alt="Mingma Sherpa" /> 

            <div class="recent-content">
             <hr>
             <div class="clearfix"></div>
             <a href="media-gallery.php"><h2>Video Posts</h2> </a>   
           </div>
         </div>
       </div>

       <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
        <div class="recent-block">
          <img src="images/activity/3.jpg" class="img-responsive responsive--full wp-post-image" alt="Mingma Sherpa" /> 

          <div class="recent-content">
            <hr>
            <div class="clearfix"></div>
            <a href="media-gallery.php"><h2>Personal Blogs</h2> </a>
          </div>
        </div>
      </div>
          
      <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
          <div class="recent-block">
            <img src="images/activity/1.jpg" class="img-responsive responsive--full wp-post-image" alt="Mingma Sherpa" /> 
            <div class="recent-content">
              <hr>
              <div class="clearfix"></div>
                <a href="media-gallery.php"><h2>Media Coverage</h2> </a>      
            </div>
          </div>
        </div>

        <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
          <div class="recent-block">
            <img src="images/activity/2.jpg" class="img-responsive responsive--full wp-post-image" alt="Mingma Sherpa" /> 

            <div class="recent-content">
             <hr>
             <div class="clearfix"></div>
             <a href="media-gallery.php"><h2>Video Posts</h2> </a>   
           </div>
         </div>
       </div>

       <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
        <div class="recent-block">
          <img src="images/activity/3.jpg" class="img-responsive responsive--full wp-post-image" alt="Mingma Sherpa" /> 

          <div class="recent-content">
            <hr>
            <div class="clearfix"></div>
            <a href="media-gallery.php"><h2>Personal Blogs</h2> </a>
          </div>
        </div>
      </div>
 </div>
</div>
</section>
<!-- /gallery Section -->
<?php include_once('footer.php'); ?>
