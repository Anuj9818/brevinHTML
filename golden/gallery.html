<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
<title>Home | Our Gallery</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="FontAwesome 4.1.0/css/font-awesome.css"/>
<!--lightbox-->
	<link rel="stylesheet" type="text/css" href="lightbox/css/lightbox.css">
 
<!--lightbox -->
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />
<link href="css/owl.carousel.css" rel="stylesheet">
<link href="css/jquery.bxslider.css" rel="stylesheet">

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

<!--logo slider Starts -->

<!--logo slider ends -->
<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700,800" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
</head>

<body>
 
  <header class="header"><!--//.header starts-->
    <div class="top-bar">
      <div class="container">
        <ul class="social-icons">
          <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
          <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
          <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
          <li><a href="#"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
        </ul>
      </div>
    </div>
    <div id="pav-mainnav">
      <div class="navigation-bar">
        <div class="container">
          <div class="row">
            <div class="col-sm-4">
              <figure class="logo_holder"> <a href="index.html"><img src="images/main-logo.png"></a> </figure>
            </div>
            <div class="col-sm-8 nav_col">
              <div class="navbar" role="navigation"> 
                <!--navbar-->
                <div class="navbar-header navbar-inverse">
                  <h4>Menu</h4>
                  <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
                </div>
                <div class="collapse navbar-collapse"><!--nav-collapse-->
                  
                  <ul class="nav navbar-nav">
                    <li><a href="index.html">Home </a></li>
                    <li><a href="about_us.html" > About Us </a></li>
                    <li><a href="our_services.html"> Our Services </a> </li>
                    <li><a href="our_clients.html"> Our Clients </a></li>
                    <li class="active"><a href="gallery.html"> Gallery</a></li>
                    <li><a href="contact_us.html" > Contact Us</a></li>
                  </ul>
                </div>
              </div>
            </div>
            <!-- /.nav-collapse --> 
          </div>
        </div>
      </div>
    </div>
  </header>
  <!--//.header ends--> 
  
 

<!--.//.MAIN SLIDER STARTS-->
<section class="banner">

    <figure class="about-banner">
     <img src="images/gallery-banner.jpg"  alt="" title=""> 
     </figure>
<div class="breadCrumbNav">
<div class="container">
<h2 class="page_title">Our Gallery</h2>
</div>

</div>
 

</section>
<!--.//.MAIN SLIDER ENDS--> 


 

<!-- //GALLERY CONTENT STARTS-->

 <section class="content-holder gallery_content page_holder">
  <div class="container">
 
 
	<div class="image-set row">
            <div class="col-sm-4 col-xs-6"> 
        
					<a class="example-image-link" href="lightbox/img/image-1.jpg" data-lightbox="example-set" data-title="Click the right half of the image to move forward.">
                    
                        <figure>
                    <img class="example-image" src="lightbox/img/image-1.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    </a>
 </div>                
 
         <div class="col-sm-4 col-xs-6">  
					<a class="example-image-link" href="lightbox/img/image-2.jpg" data-lightbox="example-set" data-title="Or press the right arrow on your keyboard.">
                    
                            <figure>
                    <img class="example-image" src="lightbox/img/image-2.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    </a>
                    </div>
                    
                             <div class=" col-sm-4 col-xs-6"> 
					<a class="example-image-link" href="lightbox/img/image-3.jpg" data-lightbox="example-set" data-title="The next image in the set is preloaded as you're viewing.">
                          <figure>
                    <img class="example-image" src="lightbox/img/image-3.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    </a>
                    </div>
         
 
            <div class="col-sm-4 col-xs-6"> 
					<a class="example-image-link" href="lightbox/img/image-4.jpg" data-lightbox="example-set" data-title="Click the right half of the image to move forward.">
                            <figure>
                    <img class="example-image" src="lightbox/img/image-4.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    </a>
 </div>                
 
         <div class="col-sm-4 col-xs-6">  
					<a class="example-image-link" href="lightbox/img/image-5.jpg" data-lightbox="example-set" data-title="Or press the right arrow on your keyboard.">
                     <figure>
                    <img class="example-image" src="lightbox/img/image-5.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    </a>
                    </div>
                    
                             <div class="col-sm-4 col-xs-6"> 
					<a class="example-image-link" href="lightbox/img/image-6.jpg" data-lightbox="example-set" data-title="The next image in the set is preloaded as you're viewing.">
                           <figure>
                    <img class="example-image" src="lightbox/img/image-6.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    </a>
                    </div>
 
            <div class="col-sm-4 col-xs-6"> 
					<a class="example-image-link" href="lightbox/img/image-7.jpg" data-lightbox="example-set" data-title="Click the right half of the image to move forward.">
                          <figure>
                    <img class="example-image" src="lightbox/img/image-7.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    </a>
 </div>                
 
         <div class="col-sm-4 col-xs-6">  
					<a class="example-image-link" href="lightbox/img/image-8.jpg" data-lightbox="example-set" data-title="Or press the right arrow on your keyboard.">
                           <figure>
                    <img class="example-image" src="lightbox/img/image-8.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    
                    </a>
                    </div>
                    
                             <div class="col-sm-4 col-xs-6"> 
					<a class="example-image-link" href="lightbox/img/image-9.jpg" data-lightbox="example-set" data-title="The next image in the set is preloaded as you're viewing.">
                          <figure>
                    <img class="example-image" src="lightbox/img/image-9.jpg" alt="" title="Click here to zoom" />
                    </figure>
                    
                    </a>
                    </div>
                </div>
     
     
     
     
     
  </div>
</section>
<!-- //GALLERY CONTENT ENDS--> 

 

<section class="content-holder footer">
  <div class="container">
    <div class="row inner_content">
      <div class="col-sm-3 col-xs-12">
        <h4 class="title">Info</h4>
        <ul class="page_links sameheight3">
          <li><a href="index.html">Home </a></li>
          <li><a href="about_us.html">About Us </a></li>
          <li><a href="our_services.html">Our Services </a></li>
          <li><a href="our_clients.html">Our Clients </a></li>
          <li><a href="news.html">News & Updates </a></li>
          <li><a href="gallery.html">GAllery </a></li>
          <li><a href="contact_us.html">Contact US </a></li>
        </ul>
      </div>
      <div class="col-md-offset-1 col-md-3 col-sm-offset-1 col-sm-4 col-xs-12">
        <h4 class="title">Contact Us</h4>
        <div class="contact_wrapper sameheight3">
          <address>
          <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
          <div class="details"> <span> Lalitpur,Kupondole<br>
           Kathmandu </span> </div>
          </address>
          <address>
          <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
          <div class="details"> <span> <a href="tel:015555039" class="linkto">015555039 </a> </span> </div>
          </address>
          <address>
          <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
          <div class="details"> <span> <a href="mailto:info@goldentriangle.com.np" class="linkto">info@goldentriangle.com.np</a> </span> </div>
          </address>
        </div>
      </div>
      <div class="col-md-offset-1 col-md-3 col-sm-4 col-xs-12">
        <h4 class="title">Get Social</h4>
        <ul class="social-icons sameheight3">
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Facebook"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Twitter"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Google-plus"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
        </ul>
      </div>
    </div>
    <div class="copyright">
      <p>© Copyright © 2018. Golden Triangle. All Right Reserved.</p>
    </div>
  </div>
</section>
<section class="back-top">
  <div id="back-top" class="uparrow" style="display: block;"> <a href="#top" title="Go to the top of the page"> <i class="fa fa-angle-up"></i> </a> </div>
</section>
</body>
<script src="js/jquery-1.9.1.min.js"></script>
<!--./testimonial slider starts-->
 
 
<!--./testimonial slider ends-->
<script src="js/fixed-nav.js"></script>
<script src="js/jquery.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/Push-up.js"></script>
 

<!--..//SAME HEIGHT JQUERY STARTS-->
<script type="text/javascript" src="js/sameheight.js">
</script>


<!--==/lightbox jqery-->
<script src="lightbox/js/lightbox.js"></script>
<!--==/lightbox jqery-->

<!--..//SOCIAL ICON HOVER TOOLTIP STARTS-->
<script>
$(document).ready(function(){
    $('[data-toggle="tooltip"]').tooltip();   
});
</script>
</html>
